opencensus/exporters/trace/stackdriver/internal/stackdriver_exporter.cc

```diff
@@ -277,11 +280,23 @@ void Handler::Export(
 
 }  // namespace
 
-void StackdriverExporter::Register(absl::string_view project_id) {
+grpc::Status StackdriverExporter::Register(
+    absl::string_view project_id, const StackdriverExporterOptions& opts) {
```

Comment:
project_id should be part of the options.

See for example java:
https://github.com/census-instrumentation/opencensus-java/blob/master/exporters/trace/stackdriver/src/main/java/io/opencensus/exporter/trace/stackdriver/StackdriverTraceConfiguration.java

Or in Go:
https://github.com/census-ecosystem/opencensus-go-exporter-stackdriver/blob/master/stackdriver.go#L73